src/overlays/actors/ovl_En_Dodojr/z_en_dodojr.c

```diff
@@ -178,40 +178,40 @@ void func_809F6A20(EnDodojr* this) {
     this->actor.velocity.z = 0.0f;
     this->actor.gravity = -0.8f;
 
-    if (this->unk_1FC == 0) {
-        this->unk_1FC = 3;
+    if (this->bounceCounter == 0) {
+        this->bounceCounter = 3;
         this->actor.velocity.y = 10.0f;
     }
 }
 
-void func_809F6AC4(EnDodojr* this) {
+void EnDodojr_PlayTwitchingAnimation(EnDodojr* this) {
```

Comment:
This function is not so different from `EnDodojr_SetupJumpAttackBounce` for example, but it's `_Play` vs `_Setup`
I'm not sure which is better (it doesn't really look like a "setup" but idk maybe?), but it seems like it should be consistent?

Reply:
Oh you are right they are almost the same, I think I was led astray by usage; agree it should be consistent. I felt that I was trying to shoehorn the "Setup" pattern into this actor when it doesn't really seem to follow it.

Reply:
I would say that these are still setup functions. Given the grouping in the file, and the things they accomplish, its the usual "setup" pattern but they chose not to set the action function in the setup for some reason. they do it after running the setup